msbrainpy/map.py

```diff
@@ -179,3 +180,162 @@ def affineTransform(parameters, COR, points):
         output.append([output_row])
     output = np.concatenate(output)
     return output
+
+
+# ------------------------------------------ 2D Mapping Related Functions ----------------------------------------------
+
+# ---------------------------------------- Find approximate in situ volume ---------------------------------------------
+# required for aligning images to the Allen Brain Atlas
+# ensure files for a single brain are stored in a directory
+def find_volume_coordinates(directory, out_directory, image_name_pattern=r'image_id-\d*.jpeg', scale_factor=8):
+    """
+    directory: directory in which to find rgb images
+    out_directory: directory to which tissue mask images should be saved
+    """
+    find_gene_series_masks(directory, out_directory, image_name_pattern=image_name_pattern)
+    volume_assembler = get_volume_assembler(out_directory, scale_factor=scale_factor)
+    return volume_assembler
```

Comment:
Like this thing. It would be way easier for me to understand this code if there was a "Returns" section in the docstring. =)